repository/config.go

```diff
@@ -0,0 +1,67 @@
+package repository
+
+import (
+	"context"
+
+	"release-manager/pkg/dberrors"
+	"release-manager/repository/model"
+	svcmodel "release-manager/service/model"
+
+	"github.com/nedpals/supabase-go"
+)
+
+type ConfigRepository struct {
+	client *supabase.Client
+	entity string
+}
+
+func NewConfigRepository(c *supabase.Client) *ConfigRepository {
+	return &ConfigRepository{
+		client: c,
+		entity: "config",
+	}
+}
+
+func (r *ConfigRepository) Update(ctx context.Context, c svcmodel.Config) error {
+	data := model.ToConfig(
+		c.OrganizationName,
+		c.DefaultReleaseMessage,
+		c.Slack,
+		c.Github,
+	)
+
+	err := r.client.
+		DB.From(r.entity).
+		Upsert(&data).
```

Comment:
I've realized that it's more practical to pass data for updates and inserts as pointers, especially when a custom `MarshalJSON` function is implemented. To maintain consistency, I've applied this change to all insert and update operations.

`model.ToConfig()` could return a pointer directly, but I believe it is better to show the intention of passing pointer more explicitly.